Modules/posixmodule.c

```diff
@@ -8696,7 +8711,39 @@ _fdwalk_close_func(void *lohi, int fd)
     }
     return 0;
 }
-#endif /* HAVE_FDWALK */
+#endif /* USE_FDWALK */
+
+/* Closes all file descriptors in [first, last], ignoring errors. */
+void
+_Py_closerange(int first, int last)
+{
+#ifdef USE_FDWALK
+    int lohi[2];
+#endif
```

Comment:
This declaration can be moved down, eliminating the need for this extra `#ifdef`.

Reply:
Ah, thanks! I missed this part: `This means (amongst many other things) that all declarations must be at the top of a block (not necessarily at the top of function).` ->